recipes/samarium/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.16)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+# find_package(samarium CONFIG REQUIRED)
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::samarium)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 20)
```

```suggestion
find_package(SAMARIUM CONFIG REQUIRED)

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} samarium::samarium)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 20)
```
CONAN_PKG:: will be deprecated. Dont forget to use the generator cmake_find_package_multi, so Conan will create SAMARIUM-config.cmake